packages/orbit-components/.storybook/orbitDecorator.tsx

```diff
@@ -15,8 +16,18 @@ const OrbitDecorator: Decorator = (storyFn, context) => {
   React.useEffect(() => {
     const html = document.querySelector("html");
     if (html && !html.getAttribute("dir")) html.setAttribute("dir", "ltr");
+    const componentForceRerender = () => {
+      // Invokes Storybook's addon API method event to trigger a UI refresh.
+      // Ensures that a11y addons is refreshed and shows the correct list of component accessibility issues.
+      console.log("Component is being rerendered.");
+      addons.getChannel().emit("forceReRender");
```

Comment:
An explanation of the function is [described here](https://gitlab.skypicker.com/frontend/core/-/merge_requests/1784#note_4983189).